docker-compose/README.md

```diff
@@ -0,0 +1,35 @@
+# docker-compose
+
+The docker-compose environment consists of:
+
+- a standalone Flink cluster (`jobmanager` and `taskmanager` containers).
+- an "edge" node from which Flink jobs can be scheduled (`flink-sql-runner-node` container).
+- minio instance which serves as AWS S3 substitute.
+- a short-living container `createbuckets` which configures minio, namely, creates necessary buckets, policies and
+  users.
+
+Flink UI can be accessed at `http://localhost:8081`.
+Minio UI can be accessed at `http://localhost:9001` (`user`/`password`).
+
+---
+
```

Comment:
I would add a command for running docker-compose to set up the environment before running the example.
`docker-compose -f docker-compose/docker-compose.yaml up -d`

Reply:
done